### PR TITLE
Update addons channel description.

### PR DESCRIPTION
**Pull request by brekk**
Issue:

Existing documentation is confusing / incomplete.

## What I did

Added a description.

## How to test

Reading. 😉 

_Is this testable with jest or storyshots?_
Nah.

_Does this need a new example in the kitchen sink apps?_
Prolly not.

_Does this need an update to the documentation?_
It _is_ an update to the documentation.

Hey there,

Please let me know if there's more needed here, I ran into this this morning and I felt the existing documentation was lacking.

Thanks!

### CODE REVIEW

**Comment by Hypnosphi** [comment]
Thank you! In my view, it should be an explanation of point 1, rather than a separate point

**Review by Hypnosphi — APPROVED** [review]
LGTM, but I'd appreciate a second pair of eyes

**Review by danielduan — APPROVED** [review]
100% thanks!

Kind of hesitant to say the same version because we don't publish new versions of addons that didn't have any changes since the last version. There's a good chance that a version number for a specific addon wasn't published.

File: docs/pages/basics/faq/index.md [hunk 1]
```diff
@@ -27,8 +27,10 @@ You can generally reuse webpack rules fairly easily by placing them in a file th
 ### Why is there no addons channel?
 
 A common error is that an addon tries to access the "channel", but the channel is not set. This can happen in a few different cases:
-1\. In storybook/addon development, it can be an NPM version problem. If there are two versions of the addons NPM package, it will cause problems. In this case, make sure there is only a single version of `@storybook/addons` being used by your project.
-2\. In React Native, it's a special case that's documented in [#1192](https://github.com/storybooks/storybook/issues/1192)
+
+1. In storybook/addon development, it can be an NPM version problem. If there are two versions of the addons NPM package, it will cause problems. In this case, make sure there is only a single version of `@storybook/addons` being used by your project.
+2. In React Native, it's a special case that's documented in [#1192](https://github.com/storybooks/storybook/issues/1192)
+3. You have inadvertently installed multiple addons with differing versions. Upgrade or downgrade all `@storybook/addon-*` modules so they have the same version.
```
Comment (danielduan) [inline docs/pages/basics/faq/index.md]: Actually, can you add something at the end of this?

Usually, deleting the node_modules and all lock files and reinstalling fixes the problem.

**Comment by Hypnosphi** [comment]
> we don't publish new versions of addons that didn't have any changes since the last version

@shilman was going to change that

**Comment by shilman** [comment]
@Hypnosphi @danielduan yes, in future versions will force publish all packages regardless of whether there have been changes to the package.

**Review by shilman — DISMISSED** [review]
Thanks @brekk !  I agree with @danielduan 's request above. A very common solution is to reinstall (as unsatisfying as that is!)

**Comment by danielduan** [comment]
I added the line. Gonna merge this.

**Comment by brekk** [comment]
@danielduan Hey thanks for the merge! I'm sorry, got bogged down with other things this week.

**Pull request merged**